numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -36,6 +36,16 @@
 #include <limits.h>
 #include <assert.h>
 
+/* check for sequences, but ignore the types numpy considers scalars */
+static NPY_INLINE npy_bool
+MyPySequence_Check(PyObject *op) {
```

Comment:
Instead of this, perhaps call it `PySequence_NoString_Check`? (Think it is fine not to mention the array scalar - that more obviously is not a sequence.)

Reply:
Name fixed